web/rest/user/config/api/raw/provider.yml

```diff
@@ -0,0 +1,171 @@
+########################################
+## Raw
+########################################
+
+Ivoz\Provider\Domain\Model\CallForwardSetting\CallForwardSetting:
+  itemOperations:
+    post_my_call_forward_settings:
+      access_control: '"ROLE_COMPANY_USER" in roles'
+      method: 'POST'
+      path: '/my/call_forward_settings'
+      route_name: 'post_my_call_forward_settings'
+      swagger_context:
+        consumes: ["application/ld+json"]
+        produces:
+        - 'application/json'
+        - 'application/ld+json'
+        tags:
+        - My
+        parameters:
+        - name: 'CallForwardSetting'
+          in: 'body'
+          description: 'The updated CallForwardSetting resource'
+          schema:
+            $ref: '#/definitions/CallForwardSetting'
+  collectionOperations:
+    get_my_call_forward_settings:
+      depth: 1
+      access_control: '"ROLE_COMPANY_USER" in roles'
+      method: 'GET'
+      path: '/my/call_forward_settings'
+      route_name: 'get_my_call_forward_settings'
+      operation_normalization_context: 'detailedCollection'
+      normalization_context:
+        groups: ['detailedCollection']
+      swagger_context:
+        parameters: []
+        produces:
+        - 'application/json'
+        - 'application/ld+json'
+        tags:
+        - My
+  attributes:
+    access_control: '"ROLE_COMPANY_USER" in roles'
+
+Ivoz\Provider\Domain\Model\Country\Name:
+Ivoz\Provider\Domain\Model\Country\Zone:
+Ivoz\Provider\Domain\Model\Country\Country:
+  itemOperations:
+    get_my_company_country:
+      access_control: '"ROLE_COMPANY_USER" in roles'
+      method: 'GET'
+      path: '/my/company_country'
+      route_name: 'get_my_company_country'
+      operation_normalization_context: 'detailed'
+      normalization_context:
+        groups: ['detailed']
+      swagger_context:
+        produces:
+        - 'application/json'
+        - 'application/ld+json'
+        tags:
+        - My
+        parameters: []
+  collectionOperations: []
+  attributes:
+    pagination_client_enabled: true
+    access_control: '"ROLE_COMPANY_USER" in roles'
+
+Ivoz\Provider\Domain\Model\Extension\Extension:
+  itemOperations: []
+  collectionOperations:
+    get_my_company_extensions:
+      method: 'GET'
+      path: '/my/extensions'
+      route_name: 'get_my_company_extensions'
+      normalization_context:
+      - collection
+      swagger_context:
+        produces:
+        - 'application/json'
+        - 'application/ld+json'
+        tags:
+        - My
+        parameters: []
+  attributes:
+    access_control: '"ROLE_COMPANY_USER" in roles'
+    swagger_context:
+      required:
+      - company
+
+Ivoz\Provider\Domain\Model\Feature\Feature:
+  itemOperations: []
+  collectionOperations: []
+
+Ivoz\Provider\Domain\Model\RatingPlanGroup\RatingPlanGroup:
+  attributes:
+    access_control: 'FALSE'
+    pagination_enabled: false
+  itemOperations: []
+  collectionOperations: []
```

Comment:
This represent a group of Destination Rates for the billing engine, not sure why it's required in User API

Reply:
removed